### PR TITLE
Disable ConnectWithRevocation_StapledOcsp on RHEL and CentOS 7

### PR DESCRIPTION
**Pull request by vcsjones**
Best as I can tell, the test is only failing on RHEL7 and CentOS 7, so let's disable it there for now.

Contributes to https://github.com/dotnet/runtime/issues/71037.

### CODE REVIEW

**Review by stephentoub — APPROVED** [review]

File: src/libraries/System.Net.Security/tests/FunctionalTests/CertificateValidationRemoteServer.cs [hunk 1]
```diff
@@ -101,12 +101,17 @@ public Task ConnectWithRevocation_WithCallback(bool checkRevocation)
...
 
         [PlatformSpecific(TestPlatforms.Linux)]
-        [Theory]
+        [ConditionalTheory]
         [OuterLoop("Subject to system load race conditions")]
         [InlineData(false)]
         [InlineData(true)]
         public Task ConnectWithRevocation_StapledOcsp(bool offlineContext)
         {
+            if (PlatformDetection.IsRedHatFamily7)
```
Comment (bartonjs) [inline src/libraries/System.Net.Security/tests/FunctionalTests/CertificateValidationRemoteServer.cs:110]: The issue says it's only failing when offlineContext is false.  Should this only skip when it's false?

Comment (vcsjones) [reply src/libraries/System.Net.Security/tests/FunctionalTests/CertificateValidationRemoteServer.cs]: Seems reasonable.

**Review by bartonjs — APPROVED** [review]
Thanks for jumping on this.

**Review by wfurt — APPROVED** [review]
LGTM

**Comment by wfurt** [comment]
no failure in ConnectWithRevocation_StapledOcsp.

**Pull request merged**